### PR TITLE
Validate v7r3p2

### PR DESCRIPTION
**Pull request by atsareg**

### CODE REVIEW

**Comment by atsareg** [comment]
The AccountingDB integration tests are not compatible by the fix in the grouping logic in the retrieveBucketedData. This needs more investigation in how retrieveBucketedData is used

**Comment by fstagni** [comment]
> The AccountingDB integration tests are not compatible by the fix in the grouping logic in the retrieveBucketedData. This needs more investigation in how retrieveBucketedData is used

cc @arrabito 

Now the produced query is 
```
SELECT SUM(`ac_bucket_dirac-JenkinsSetup_Pilot`.`Jobs`) FROM `ac_bucket_dirac-JenkinsSetup_Pilot`, `ac_key_dirac-JenkinsSetup_Pilot_Site` WHERE `ac_bucket_dirac-JenkinsSetup_Pilot`.`startTime` >= 1261612800 AND `ac_bucket_dirac-JenkinsSetup_Pilot`.`startTime` <= 1262217600 AND `ac_bucket_dirac-JenkinsSetup_Pilot`.`Site` = `ac_key_dirac-JenkinsSetup_Pilot_Site`.`id` GROUP BY  ORDER BY `ac_bucket_dirac-JenkinsSetup_Pilot`.`Site`
```

clearly what's wrong is that there's nothing in `GROUP BY`.

**Comment by atsareg** [comment]
Yes, this is what is wrong in the SQL. But the test does not pass a field used in grouping also in the select list of things, which looks like a logical error.

File: src/DIRAC/AccountingSystem/DB/AccountingDB.py [hunk 1]
```diff
@@ -1149,7 +1151,16 @@ def __queryType(
...
     if groupFields:
-      cmd += " GROUP BY %s" % (groupFields[0] % tuple(groupFields[1]))
+      if len(groupFields[1]) == 1:
+        testGroupFields = " %s" % selectFields[0] % tuple(realFieldList)
+        testGroupFieldsList = testGroupFields.split(",")
+        realGroupFields = ()
+        for testGroupFields in testGroupFieldsList:
+          if "sum" not in testGroupFields.lower():
+            realGroupFields += (testGroupFields.strip(),)
+        cmd += " GROUP BY " + ','.join(realGroupFields)
```
Comment (TaykYoku) [inline src/DIRAC/AccountingSystem/DB/AccountingDB.py:1161]: IIUC `realGroupFields` can be empty
```suggestion
        if realGroupFields:
          cmd += " GROUP BY " + ','.join(realGroupFields)
```

**Pull request merged**